classes/mfgtool-initramfs-image.bbclass

```diff
@@ -12,7 +12,7 @@ FEATURE_PACKAGES_mtd = "packagegroup-fsl-mfgtool-mtd"
 FEATURE_PACKAGES_extfs = "packagegroup-fsl-mfgtool-extfs"
 FEATURE_PACKAGES_f2fs = "packagegroup-fsl-mfgtool-f2fs"
 
-IMAGE_FSTYPES = "cpio.gz.u-boot"
+IMAGE_FSTYPES = "cpio.xz.u-boot"
```

Comment:
For space-sensitive environments, `zstd` seems to be better as it uses less memory and has better performance. See if it is an option.

Reply:
I did a grep and found an interesting line here:

https://github.com/Freescale/meta-freescale/blob/master/classes/kernel-itbimage.bbclass#L386

Do I need to add `cpio.zst`?

Reply:
Yes. :-D

Reply:
Unfortunately `zstd` size improvement is much more modest:

```
 18M fsl-image-mfgtool-initramfs-imx8mqevk-20220322203449.rootfs.cpio.gz.u-boot
8.1M fsl-image-mfgtool-initramfs-imx8mqevk-20220322203449.rootfs.cpio.xz.u-boot
 16M fsl-image-mfgtool-initramfs-imx8mqevk-20220322203449.rootfs.cpio.zst.u-boot
```

I'm told some argument might help, so I'll have to dig some more...

Reply:
Unfortunately `zstd` size improvement is much more modest:

```
 18M fsl-image-mfgtool-initramfs-imx8mqevk-20220322203449.rootfs.cpio.gz.u-boot
8.1M fsl-image-mfgtool-initramfs-imx8mqevk-20220322203449.rootfs.cpio.xz.u-boot
 16M fsl-image-mfgtool-initramfs-imx8mqevk-20220322203449.rootfs.cpio.zst.u-boot
```

I'm told some argument might help, so I'll have to dig some more...

Reply:
Yes, you can use up to level 22 AFAIK.